### DISCUSSION TITLE
How to get pid to interupt solving

### DISCUSSION THREAD

**Discussion by H4rryK4ne** [coin-or/pulp#757] [Q&A]
I have an application which starts solving a problem in a separate process. 

I want to provide a possibility to stop the process. I found, that I can send a SIGINT signal to the process ID of the solver (at least when using the CBC) to stop the solving process and get an intermediate result, but I don't know how to get the process ID of the solver at runtime.

**Comment by pchtsp** [comment]
If you interrupt the cbc process I don't think you will be able to load the
solution afterwards, I think. Maybe ask if that's possible at the the cbc
github.

https://github.com/coin-or/Cbc

**Reply by H4rryK4ne** [reply]
When I know the process ID I can send a SIGINT signal to the cbc process and it will stop calculating in an orderly way: 
> Result - User ctrl-cuser ctrl-c

Actually, it is not so important, that there is any result ... it is more important for to be able to stop the running calculation (e.g. the user made a mistake during input and wants to start a new calculation). But I don't know how I can get the respective process ID.
